OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -35,6 +35,9 @@ public class ProductionQueueInfo : ITraitInfo
 		[Desc("Should the prerequisite remain enabled if the owner changes?")]
 		public readonly bool Sticky = true;
 
+		[Desc("Should right clicking on the icon instantly cancel the production instead of putting it on hold?")]
+		public readonly bool DisallowOnHold = false;
```

Comment:
Can we please call this `DisallowPaused` for consistency with the rest of the code?